mappings/net/minecraft/client/gui/Element.mapping

```diff
@@ -1,38 +1,148 @@
 CLASS net/minecraft/class_364 net/minecraft/client/gui/Element
+	COMMENT Base GUI interface for handling callbacks related to
+	COMMENT keyboard or mouse actions.
+	COMMENT
+	COMMENT Mouse coordinate is bounded by the size of the window in
+	COMMENT pixels.
 	METHOD method_16014 mouseMoved (DD)V
+		COMMENT Callback for when a mouse move event has been captured.
+		COMMENT
+		COMMENT @param mouseX the X coordinate of the mouse
+		COMMENT @param mouseY the Y coordinate of the mouse
```

Comment:
Shouldn't these `@param` tags just be a comment on the parameter?

Reply:
Moved the doc to the correct place